code/game/machinery/computer/communications.dm

```diff
@@ -53,7 +53,7 @@
 	if(..())
 		return 1
 	if (using_map && !(src.z in using_map.contact_levels))
-		usr << "\red <b>Unable to establish a connection</b>: \black You're too far away from the station!"
+		usr << "<font color='red'> <b>Unable to establish a connection</b>: \black You're too far away from the station!</font>"

```

Comment:
\black is still a thing?

Also the `</font>` tag should be moved before the `\black` macro.